Updated the README.md

evan-stromatias opened the pull request:
This PR updates the `README.md` to include some basic info about the Green Light Services app. It's not done but it has some basic structure now that relates to our work

DocJosef commented:
@evan-stromatias I am getting a merge conflicts. Can we discuss how to resolve them? Thanks! Josef

evan-stromatias commented:
Hi @DocJosef I fixed the merge conflicts. It was the old `README_db.md` file that is now removed. I have included the relevant info to the `README.md` file.
